### PR TITLE
fixing validator errors

### PR DESCRIPTION
**Pull request by IhateTrains**
--- Error 1 of 1 ---
At <mod>\common\cb_types\WtWSMS_migration_unification_cb_types.txt [odoacer_takeover_war] (Line 446, column 1):
The string odoacer_takeover_war_desc was not found in localization.

I also localised some missing weird VIET modifiers and fixed Hadrian's Wall localisation

### CODE REVIEW

File: WTWSMS/localisation/VIET_modifiers.csv [hunk 1]
```diff
@@ -295,4 +295,11 @@ VIET_province_strong_lads;Strong Lads;;;;;;;;;;;;;x
...
 VIET_going_to_Scarborough_Fair;Scarborough Fair;;;;;;;;;;;;;x
 VIET_going_to_Scarborough_Fair_desc;This character has heard that great song, Scarborough Fair, and he or she has been enlightened by it. Onwards, as they say, to Scarborough Fair!;;;;;;;;;;;;;x
-VIET_erected_high_cross;Erected High Cross;;;;;;;;;;;x
\ No newline at end of file
+VIET_erected_high_cross;Erected High Cross;;;;;;;;;;;x
+VIET_opinion_refused_muafname;Refused muafname;;;;;;;;;;;x
+VIET_opinion_granted_small_muafname;Granted small muafname;;;;;;;;;;;x
+VIET_opinion_granted_big_muafname;Granted big muafname;;;;;;;;;;;x
+opinion_ugly_omelette_cook;Ugly omelette cook;;;;;;;;;;;x
+VIET_remove_kebal_insult;Remove Kebab insult;;;;;;;;;;;x
```
Comment (loup99) [inline WTWSMS/localisation/VIET_modifiers.csv]: This one we need to rename, otherwise it isn't in line with the PI forum rules.

Comment (IhateTrains) [inline WTWSMS/localisation/VIET_modifiers.csv]: lel, ok :D I forgot that Swedes are politically correct. It seems that this modifier isn't in use, so I removed it completely.

**Comment by loup99** [comment]
What was the bugs with the cultures? Just to check so that we don't end up with a fluo green or bright pink instead of what currently was. 😛

**Comment by IhateTrains** [comment]
The colors are in RGB format with a range of 0-1. Romano-Dacian color was  { 2.25 2 1.52 }, jurchen or sth color was { 0.795 1.1 1.65 }

**Comment by IhateTrains** [comment]
I will check these colors in a web tool

**Comment by IhateTrains** [comment]
They may look a bit different in game, but:
Romano-Dacian is ugly but acceptable  (3F3384)
the second one is pink :P (CB19A6), I will change it

**Comment by loup99** [comment]
Okay. As previously, tell me when you think this is ready for merge.

**Comment by IhateTrains** [comment]
Enough for today, you can merge.
offtopic: Have you set the date of release? How much time do we have (more or less)?

**Comment by loup99** [comment]
There is no date, given that I do not know how much time it will take to finalise the secret religious societies. The date is going to be evaluated and determined once the mod is free from crashes, as I don't want to be on a hurry and the mod isn't in a releasable state as long as those persist.

**Comment by IhateTrains** [comment]
Can I help you with the societes? I would add a basic society for every religion, just like tannerflick did.

**Comment by loup99** [comment]
Yes, you can help out. Here is the list of all the religions:
```
        aquileian
        nicene
        total_depravity
        eutychian
        binitarian
        georgian_orthodox
        insular
        pelagianism
        antinomianism
        ebionites
        marcionism
        montanist
        sethian
        valentinianism
        apollinarian
        adoptionism
        docetic
        pneumatomachians
        melchisedechianism
        patripassianism
        sabellianism
        psilanthropism
        audianistic
        arian
        monarchianism
        donatism
        priscillianism
        islamic
        abadites
        quranism
        qadariyya
        khawariji
        haruriyyah
        wahhabi
        mahdi
        salafi
        twelver
        zayid
        ismaili
        uwaisi
        qarmatian
        root_west_pagan
        root_west_pagan_reformed
        sanamahi
        sanamahi_reformed
        georgian_pagan
        illyran_pagan
        germanic_pagan
        germanic_pagan_reformed
        west_pagan
        scythian_pagan
        scythian_pagan_reformed
        semitic_pagan
        semitic_pagan_reformed
        dravidian_pagan
        dravidian_pagan_reformed
        central_african_pagan
        central_african_pagan_reformed
        gurzil_pagan
        msoura_pagan
        african_solar_pagan
        african_solar_pagan_reformed
        latin_solar_pagan
        latin_solar_pagan_reformed
        mandulis_pagan
        east_african_pagan
        assyrian_pagan
        cybelian_pagan
        mastimani
        neoplatonist
        mithraic_pagan
        zurvanist
        khurramite
        hepthal_religion
        tzadikite
        bosporan_faith
        ajivika
        carvaka
        mimamsa
        tantric
        bhakti
        sramanic
        confucianism
        taoism
```
The following commit can serve as a reference: https://github.com/loup99/WtWSMS/commit/df328e1a5397e0236b7611b7f6ce07730402e9ca

If you work from the bottom of the list the work won't conflict with what @tannerflick4 is doing.

**Comment by IhateTrains** [comment]
Strange, even the societes tannerflick4 added get errors like this:

--- Error 1 of 1 ---
At <mod>\decisions\mnm_secret_religious_societies_decisions.txt [title_decisions\secret_religions_prepare_grounds\potential\OR\AND\FROM\society_member_of] (Line 2711, column 7):
"secret_religious_society_montanist" is not a valid Society, SecretReligiousCult, or MaybeEventTargetSociety.

**Comment by loup99** [comment]
That is a false positive in the Validator, because the secret societies themselves are auto-generated.

**Pull request merged**